cmd/pr.go

```diff
@@ -66,5 +101,48 @@ func createPR() error {
 
 	fmt.Printf("Created PR %s\n", *pr.HTMLURL)
 
+	if reviewers != "" || teamReviewers != "" {
+		_, err = gc.RequestReviewersForPR(pr.GetNumber(), reviewers, teamReviewers)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func getPR(number int) error {
+	yellow := color.New(color.FgYellow)
+	green := color.New(color.FgGreen)
+	red := color.New(color.FgRed)
+
+	prp, err := gc.GetPR(number)
+	pr := *prp
```

Comment:
When `gc.GetPR` returns an error, prp is nil so maybe move `pr := *prp` after error checking.